dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterableTest.java

```diff
@@ -86,23 +85,25 @@ private void assertIterations(
   public void testHours() {
     Duration durationOfDay = Duration.ofDays(1);
     IntervalExpander dayExpander = IntervalExpander.createBasedOnDuration(durationOfDay);
-
-    Duration durationOfHour = Duration.ofDays(1);
-    IntervalExpander hourExpander = IntervalExpander.createBasedOnDuration(durationOfDay);
-
-    testIterable(
-        169,
-        ZonedIntervalIterableGenerator.forTimeUnitsUntilNow(24 * 7, durationOfDay, dayExpander));
-    testIterable(
-        8, ZonedIntervalIterableGenerator.forTimeUnitsUntilNow(7, durationOfDay, dayExpander));
-
-    testIterable(
-        1,
+    Duration durationOfHour = Duration.ofHours(1);
+    IntervalExpander hourExpander = IntervalExpander.createBasedOnDuration(durationOfHour);
+    ZonedIntervalIterable weekInHours =
+        ZonedIntervalIterableGenerator.forTimeUnitsUntilNow(24 * 7, durationOfHour, hourExpander);
```

Comment:
Let's use something like that `TimeUnit.DAYS.toHours(7);` to be more clear what we mean, like in your example with `168` days or hours.

Reply:
I prefer the constant here to minimize logic in tests, but you're right that it could be explained more clearly.
I added a clarifying comment.